database/index.js

```diff
@@ -31,6 +30,24 @@ let returnData = (id) => {
   return Information.findOne({productId: id}).exec();
 };
 
+let createInformation = (information) => {
+  return Information.insertMany([information]).exec();
+};
+
+ let updateInformation = (id, information) => {
+  return Information.findOneAndUpdate({productId: id}, {
+    $set: information
+  }).exec();
+};
+
+let deleteInformation = (id) => {
+  return Information.deleteOne({
+    productId: id
+  }).exec();
+};
 
 module.exports.returnData = returnData;
 module.exports.Information = Information;
+module.exports.createInformation = createInformation;
+module.exports.updateInformation = updateInformation;
+module.exports.deleteInformation = deleteInformation;
```

Comment:
Great job adding CRUD routes! I'm not too familiar with your service but wondering if in your createInfo. method you meant to use .insertMany vs. .insertOne? It depends on your use-case but just curious why you made that decision.